doc/networking.md

```diff
@@ -52,7 +52,44 @@ If you use Chaosnet, you may be interested in joining the Global
 Chaosnet: read more about it at https://chaosnet.net.
 
 ## DNS
-Check out this [external guide](https://its.victor.se/wiki/dqdev)
+To make ITS use DNS like a modern netizen, you need to do the following:
+
+1. Compile the handler for the DOMAIN: device, which interfaces to DNS.
```

Comment:
Explain about DQ, DQX, and DOMAIN?

Reply:
Yes, somewhere. I'm not sure where to document what really, this file seems to be "user friendly" while explaining DQ/DQXDEV and DOMAIN/DQDEV might be more low-level info?

Reply:
Info can go in INFO. :-)